mlrun/db/httpdb.py

```diff
@@ -363,6 +363,7 @@ def connect(self, secrets=None):
                 server_cfg.get("force_run_local") or config.force_run_local
             )
             config.function = server_cfg.get("function") or config.function
+            config.ce.mode = server_cfg.get("ce_mode", "")
```

Comment:
Let's move it to the top of the function
Thinking about it, it might be useful for testing purposes to allow the client to set its own ce mode and just add a warning like we do above for `namespace`.

Reply:
Please see @quaark 's comment above and decide on the wanted implementation.